cypress.config.ts

```diff
@@ -28,7 +28,7 @@ export default defineConfig({
         },
         baseUrl: 'http://localhost:8080',
         experimentalSessionAndOrigin: true,
-        specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
+        specPattern: 'cypress/e2e/*/*.{js,jsx,ts,tsx}',
```

Comment:
This change looks like a mistake? https://docs.cypress.io/guides/references/configuration#Testing-Type-Specific-Options uses `**` for folder recursion

Reply:
Oops; yes it was; I'd limited the number of test runs by changing this; evidently didn't back it out correctly.
```suggestion
        specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
```